scss/_carousel.scss

```diff
@@ -124,9 +125,20 @@
   }
 
   &:active {
-    --#{$boosted-prefix}control-bg: #{$carousel-control-icon-active-bg}; // Boosted mod
+    --#{$boosted-prefix}control-bg: #{$carousel-control-icon-active-bg};
     color: $carousel-control-color;
   }
+
+  &:focus:focus-visible,
```

Comment:
Just out of curiosity, why do we need this `&:focus:focus-visible`? In other component just having `&:focus { &[data-focus-visible-added] {` seems sufficient.